spring-cloud-aws-secrets-manager/src/main/java/io/awspring/cloud/secretsmanager/SecretsManagerPropertySource.java

```diff
@@ -77,13 +78,19 @@ public Object getProperty(String name) {
 	private void readSecretValue(GetSecretValueRequest secretValueRequest) {
 		try {
 			GetSecretValueResponse secretValueResponse = source.getSecretValue(secretValueRequest);
-			Map<String, Object> secretMap = jsonMapper.readValue(secretValueResponse.secretString(),
-					new TypeReference<Map<String, Object>>() {
-					});
+			try {
```

Comment:
There is no point to keep old try catch block here since 
`GetSecretValueResponse secretValueResponse = source.getSecretValue(secretValueRequest);`
can't throw `JsonProcessingException`